### DISCUSSION TITLE
Anonymous Issues (Don’t need to have a GitHub account)

### DISCUSSION THREAD

**Discussion by 201flaviosilva** [community/community#13270] [Projects and Issues]
Hey there,

I think it would be a good idea for projects to have an option to choose if it's possible for people who don't have a GitHub account to be able to create issues, something like Bitbucket has:

<img width="477" alt="Bitbucket" src="https://user-images.githubusercontent.com/55815265/159533516-ebe0fa86-77d9-4566-aa58-2c8b1e97627d.png">

This way it would be easier for the designer/users to be able to report issues without having to use another application to do so, and it would even have a better relationship between user and developer.

(Sorry if this already exists, and I'm too blind and didn't see it, and if it’s how? 😅)

**Comment by ghost** [comment]
Hey 201flaviosilva.

your idea is wonderful, but your request is similar to an old request here on GitHub: [Anonymous accounts in GitHub? #34595](https://github.com/community/community/discussions/34595), [(Feature request) Allow anonymous user to send issue without account #2060](https://github.com/github/hub/issues/2060)

**Comment by norwd** [comment]
I'd be concerned with this as an avenue for abuse, the barrier of entry for a GitHub account is just having an email, and there are already so many spam accounts (especially recently, see https://github.com/community/community/discussions/48334).

**Comment by DeCentN2Madness** [comment]
Those with concerns or no desire for this feature could just ignore the settings check box in repository settings if made _**opt-in** per repo_. 🤯

**Comment by brospars** [comment]
This would be great for Discussions too
